lib/internal/modules/cjs/loader.js

```diff
@@ -1082,7 +1083,17 @@ Module._initPaths = function() {
   } else {
     prefixDir = path.resolve(process.execPath, '..', '..');
   }
-  var paths = [path.resolve(prefixDir, 'lib', 'node')];
+  var postDirs = [];
+  if (relativePaths) {
+    relativePaths.split(path.delimiter).map(path => {
```

Comment:
`s/map/forEach/` or `for (const path of relativePaths.split(path.delimiter))` although an even easier way of writing this is:
```js
const postDirs = relativePaths.split(path.delimiter).filter(path => path);
```
No `if (relativePaths)` guard needed.

Reply:
Feel free to patch. see above